packages/plugins/typescript/graphql-request/src/index.ts

```diff
@@ -5,6 +5,9 @@ import { GraphQLRequestVisitor } from './visitor';
 import { extname } from 'path';
 import { RawGraphQLRequestPluginConfig } from './config';
 
+export type SdkFunctionWrapper = <T>(action: () => Promise<T>) => Promise<T>;
+export const defaultWrapper: SdkFunctionWrapper = sdkFunction => sdkFunction();
```

Comment:
thoughts on injecting this identify function into the generated files instead of exporting it? It currently adds a runtime dependency on `@graphql-codegen/typescript-graphql-request`

Reply:
oh, already fixed by #3667 👍